guides/howtos/Many to many and casting.md

```diff
@@ -0,0 +1,383 @@
+# Many to many and casting
```

Comment:
Perhaps a better name is:

* Polymorphic associations with many to many

or:

* Casting polymorphic associations with many to many

I prefer the first. What do you think?

Reply:
I like the first one too. Especially because it's shorter. It still gets part of it cut off along with a couple of children, but I'm not sure there's much we can do about it.

![image](https://user-images.githubusercontent.com/457237/59060606-db1b5c80-8877-11e9-9c30-8cc36e443fb8.png)